cos_alerter/daemon.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import datetime
+import durationpy
+import json
+import signal
+import subprocess
+import sys
+import textwrap
+import time
+
+from .alerter import DataWriter, config, send_notifications
+
+
+def notify(time_data):
+    if not time.time() - time_data.notify_time > durationpy.from_str(config['notify']['repeat_interval']).total_seconds():
```

Comment:
**suggestion (non-blocking):**

This condition is very long. Maybe the RHS would benefit from being a  var or func?

Reply:
I created `def down_interval():`.